### DISCUSSION TITLE
Next record button

### DISCUSSION THREAD

**Discussion by micooro** [YetiForceCompany/YetiForceCRM#15531] [How to]
Hi to all, 
it's possibile to have a next and previous record button on detail view? It's to hard move throught the records.

Mike

**Comment by mariuszkrzaczkowski** [comment]
it is possible to do but not efficient and problematic

**Reply by micooro** [reply]
why it'is not efficient? now i have to enter in detail and then list again to see another detail's record. If i have next and previous button i can move throught very quickly

**Reply by rskrzypczak** [reply]
but how will you know which record should be next and which should be previous?
It's not so simple.
In addition, you should also bear in mind performance issues, not only user convenience

**Reply by micooro** [reply]
"but how will you know which record should be next and which should be previous?"  Id Record????Why not?
In vtiger, this feature exist and i imaged that maybe it's apply even in Yeti
